### DISCUSSION TITLE
Component tests failing intermittently

### DISCUSSION THREAD

**Discussion by devjared1** [cypress-io/cypress#26941] [Questions and Help]
I'm using the latest Cypress (12.13.0). When I run tests locally (Windows 10) with `npx cypress run --component`, the tests will fail intermittently. I will get this error: `Failed to fetch dynamically imported module: http://localhost:3000/__cypress/src/@fs/<path_to_file>`. I have 60 spec files running, and it always only every fails on 1 spec file. All the others run and pass like they should. Here is a screenshot:<br>
![image](https://github.com/cypress-io/cypress/assets/71391311/9f46c166-be0b-48cd-b754-db3c08cf9b64)<br><br>Has anyone else experienced this error before? I've had to turn off my team's test pipeline in Gitlab because the job failed ~50% of the time.

**Comment by marktnoonan** [comment]
Making a few assumptions, but I believe this is connected to an underlying [issue in Vite, mentioned here](https://github.com/cypress-io/cypress/issues/25913#issuecomment-1579525969). It looks like the Vite PR referenced in that issue has merged. So it's likely will be able to make some progress on this soon.
